src/iconsManifest/supportedExtensions.ts

```diff
@@ -1036,7 +1036,7 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'config',
-      extensions: ['plist'],
+      extensions: ['plist', '.tool-versions'],
```

Comment:
Is `.tool-versions` a extension or a file name? In case it's a extension, you should remove the `.`. If it's a filename, you should create another entry: 
```json
{

   icon: 'config',
   extensions: '.tool-versions',
   light: true,
   filename: true,
   format: FileFormat.svg
}
```

Reply:
Note that the test are failing because you put a dot into an extension 😉

Reply:
Done